scout/server/blueprints/variant/templates/variant/buttons.html

```diff
@@ -11,7 +11,7 @@
         <select name="manual_rank" class="form-control">
           <option value="-1">Select a tag...</option>
           {% for data in manual_rank_options %}
-            <option {% if data.internal_id == variant.manual_rank %} selected {% endif %} value="{{ data.internal_id }}">
+            <option {% if data.rank == variant.manual_rank %} selected {% endif %} value="{{ data.rank }}">
```

Comment:
The unrecognized old rankings and dismissed terms was due to this. The variant updated with new manual rank and dismissed terms looks like this on master:

![image](https://user-images.githubusercontent.com/28093618/118501913-7d614180-b729-11eb-8cb5-19198723e5ab.png)